### PR TITLE
Accessibility wave 2

### PR DESCRIPTION
**Pull request by gemfarmer**
For #1366 

👀 [PREVIEW](https://federalist.18f.gov/preview/18F/doi-extractives-data/accessibility-wave-two/) 👀 

I have made several small accessibility changes to all non-explore pages.
- Glossary (labels and buttons)
- Header, several labelling and button errors
- Updated the accordion so that it is using ARIA components. Still not using the [18F accordion](github.com/18F/accordion) because it uses jquery and underscore, and I like that EITI's accordion doesn't have dependencies. Might be worth contributing to that accordion so that it doesn't require dependencies.
- implementation of new accordion

@nickbristow I am still running into some errors/warnings with HTML_CodeSniffer and WAVE telling me that labels aren't properly set up on the search bar and glossary, but I believe that they are properly labelled. Any best guesses as to why this might be happening?

### CODE REVIEW

File: js/components/accordion.js [hunk 1]
```diff
@@ -34,22 +60,90 @@
...
+          accordionItems = accordion.querySelectorAll('[accordion-item]');
+          // check if node list is different than list of children
+          if (accordionItems.length !== accordion.children.length) {
+            this.forEach(accordion.children, function(index, item) {
+              // add [accordion-item] attribute to list items
+              if (item.tagName.toLowerCase() === 'li' &&
+                !item.hasAttribute('accordion-item')) {
+                item.setAttribute('accordion-item', true);
+              }
+            });
```
Comment (gemfarmer) [inline js/components/accordion.js]: I agree that making loops within loops is generally bad practice, but definitely the most straightforward approach when I am dealing with lists of lists.

**Comment by gemfarmer** [comment]
@meiqimichelle ready for review!

**Comment by meiqimichelle** [comment]
Something is not happy on the homepage :( Firefox and Chrome.
screenshot 2016-04-22 16 14 28

**Comment by gemfarmer** [comment]
@meiqimichelle addressed tab panel issue with 4d5b29e

**Pull request merged**